tests/e2e/specs/dashboard/dashboard-noscript.test.js

```diff
@@ -0,0 +1,39 @@
+/**
+ * WordPress dependencies
+ */
+import { visitAdminPage } from '@wordpress/e2e-test-utils';
+
+describe( 'Site Kit noscript notice', () => {
+	beforeEach( async () => {
+		await visitAdminPage( 'admin.php', 'page=googlesitekit-splash' );
+	} );
+
+	describe( 'When Javascript is enabled', () => {
+		it( 'Should not display noscript notice', async () => {
+			const noscript = await page.$( '#wpbody-content' );
+			await expect( noscript ).not.toMatchElement( '.googlesitekit-noscript' );
+		} );
+	} );
+
+	describe( 'When Javascript is disabled', () => {
+		beforeAll( async () => {
```

Comment:
We discussed on the call enabling/disabling javascript within each test as it was breaking other things inside the test. I think for consistency, let's toggle JS within each test,

```
                it( 'Should not display plugin html', async () => {
                        await page.setJavaScriptEnabled( false );
			const noscript = await page.$( '#wpbody-content' );
			await expect( noscript ).not.toMatchElement( '.js-googlesitekit-plugin' );
                        await page.setJavaScriptEnabled( true );
		} );

		it( 'Should display noscript notice', async () => {
                        await page.setJavaScriptEnabled( false );
			const noscript = await page.waitForSelector( '.googlesitekit-noscript', {
				visible: true,
			} );
			await expect( noscript ).toMatchElement( '.googlesitekit-noscript__title', { text: 'The Site Kit by Google plugin requires JavaScript to be enabled in your browser.' } );
                        await page.setJavaScriptEnabled( true );
		} );
```

Reply:
This tests works out ok but I will make the change

Reply:
@nicholasio JS need to be disabled before user navigate to the the page. 

https://github.com/Shavindra/site-kit-wp/pull/3/files#diff-d24cbec8d794b4a26a34c30dd6737b77R8

Here I think it will be fine to leave it how it was

CC: @aaemnnosttv

Reply:
Gotcha, ok. @Shavindra I think we can move this to the google repo and have the official review there.